pipenv/environment.py

```diff
@@ -483,18 +483,26 @@ def activated(self, include_extras=True, extra_dists=None):
             os.environ["PYTHONIOENCODING"] = vistir.compat.fs_str("utf-8")
             os.environ["PYTHONDONTWRITEBYTECODE"] = vistir.compat.fs_str("1")
             os.environ["PYTHONPATH"] = self.base_paths["PYTHONPATH"]
+            os.environ.pop("PYTHONHOME", None)
             if self.is_venv:
                 os.environ["VIRTUAL_ENV"] = vistir.compat.fs_str(prefix)
             sys.path = self.sys_path
             sys.prefix = self.sys_prefix
             site.addsitedir(self.base_paths["purelib"])
             if include_extras:
                 site.addsitedir(parent_path)
-                sys.path.extend([parent_path, patched_dir, vendor_dir])
+                sys.path.extend([patched_dir, vendor_dir])
                 extra_dists = list(self.extra_dists) + extra_dists
                 for extra_dist in extra_dists:
                     if extra_dist not in self.get_working_set():
                         extra_dist.activate(self.sys_path)
+            # Move the added paths to the front
+            new_paths = []
+            for item in list(sys.path):
```

Comment:
You'll need to do some normalization here, also I'm not totally sure you need to put things at the front.  You can probably just do `sys.path = list(set(sys.path) | set(original_path))` (why is this necessary though?)

Reply:
Oh, it is just taken from `activate_this.py`, where I think virtualenv paths should always take priority to other paths.